lib/spaceship/tunes/tunes_client.rb

```diff
@@ -348,6 +348,32 @@ def update_app_version!(app_id, version_id, data)
     end
 
     #####################################################
+    # @!group AppAnalytics
+    #####################################################
+
+    def time_series_analytics(app_ids, measures, start_time, end_time, frequency)
```

Comment:
The underlying request has more parameters.

So maybe the method parameters should have the following style instead:

```app_ids: nil, ....```

to allow for changes in the future.